guides.md

```diff
@@ -0,0 +1,125 @@
+ - [Setting up Kogito CLI to deploy Kogito services on OpenShift](https://docs.jboss.org/kogito/release/latest/html_single/#proc_kogito-vscode-extension)
+
+### Kogito on OpenShift
+ - [Deploying Kogito services on OpenShift](https://docs.jboss.org/kogito/release/latest/html_single/#chap_kogito-deploying-on-openshift )
+ - [Using the Kogito Operator (Coming soon)]()
+
+### Decision services
+ - [Developing decision services with Kogito](https://docs.jboss.org/kogito/release/latest/html_single/#chap_kogito-developing-decision-services)
+ - [Using DMN models in Kogito services](https://docs.jboss.org/kogito/release/latest/html_single/#_using_dmn_models_in_kogito_services)
+ - [Using DRL rules in Kogito services](https://docs.jboss.org/kogito/release/latest/html_single/#_using_drl_rules_in_kogito_services)
+ - [Using spreadsheet decision tables in Kogito services](https://docs.jboss.org/kogito/release/latest/html_single/#_using_spreadsheet_decision_tables_in_kogito_services)
+ - [Drools engine in Kogito](https://docs.jboss.org/kogito/release/latest/html_single/#_drools_engine_in_kogito)
+
+### Process services
+ - [Developing process services with Kogito (Coming soon)]()
+ - [BPMN 2.0 support in Kogito (Coming soon)]()
+
+### Environment configuration
+ - [Using the Kogito Data Index Service (Coming soon)]()
+ - [Using the Kogito Jobs Service (Coming soon)]()
+ - [Kogito runtime confirmation (Coming soon)]()
+
+</div>
+<div class="grid__item width-8-12 width-12-12-m gs-content">
+
+You have gone through the <a href="{{site.baseurl}}/get-started">Getting Started guides</a>. You are now ready to dive into specific topic guides.
+
+<h2 id="Tooling">Tooling</h2>
+
+<div class="guide-item" markdown="1">
+### Enabling the Kogito VSCode extension for BPMN and DMN modeling
+
+Visual Studio Code (VSCode) is the preferred integrated development environment (IDE) for developing Kogito services. Kogito provides a VSCode extension that enables you to design Business Process Model and Notation (BPMN) 2.0 business processes and Decision Model and Notation (DMN) decision models directly in VSCode.
+
+<a href="https://docs.jboss.org/kogito/release/latest/html_single/#proc_kogito-vscode-extension" class="button-cta secondary">READ THE GUIDE</a>
+</div>
+
+<div class="guide-item" markdown="1">
+### Setting up Kogito CLI to deploy Kogito services on OpenShift
+
+The Kogito command-line interface (CLI) enables you to interact with the Kogito Operator for deployment tasks.
+
+<a href="https://docs.jboss.org/kogito/release/0.8.0/html_single/#proc_kogito-deploying-on-ocp-kogito-cli" class="button-cta secondary">READ THE GUIDE</a>
+</div>
+
+<h2 id="Kogito_on_OpenShift">Kogito on OpenShift</h2>
+
+<div class="guide-item" markdown="1">
+### Deploying Kogito services on OpenShift using the OpenShift web console
+
+After you create your Kogito services as part of a business application, you can use the OpenShift web console to deploy your services.
```

Comment:
after creating your..?

Reply:
@spolti , just less clear, in terms of "global English style" conventions. Plus would introduce an inconsistency with the rest, in the docs anyway.